desktop/apps/auction2/actions.js

```diff
@@ -75,11 +88,49 @@ export function fetchArtworks() {
   }
 }
 
+export function fetchArtworksByFollowedArtists() {
+  return async (dispatch, getState) => {
+    const {
+      auctionArtworks: {
+        followedArtistRailMax,
+        followedArtistRailPage,
+        filterParams,
+        user
+      }
+    } = getState()
+
+    const inputVars = {
+      sale_id: filterParams.sale_id,
+      page: followedArtistRailPage,
+      size: followedArtistRailMax
+    }
+
+    try {
+      const { filter_sale_artworks } = await metaphysics({
+        query: worksByFollowedArtists,
+        variables: inputVars,
+        req: {
+          user
+        }
+      })
+      if (filter_sale_artworks.hits.length > 0) {
+        dispatch(updateSaleArtworksByFollowedArtists(filter_sale_artworks.hits))
+        dispatch(updateSaleArtworksByFollowedArtistsTotal(filter_sale_artworks.counts.total))
+        dispatch(updateIsLastFollowedArtistsPage())
+        dispatch(showFollowedArtistsRail())
+      }
+    } catch(error) {
+      console.error('error!', error)
```

Comment:
Maybe, but these could be errors in any part of the system, so I think for now the fact that it renders 0 artworks + the console indication is okay.